account/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+class User(models.Model):
+    login_id     = models.CharField(max_length=100)
+    password     = models.CharField(max_length=100)
+    name         = models.CharField(max_length=20)
+    phone_number = models.CharField(max_length=11)
+    create_date  = models.DateField(auto_now_add=True)
+    email        = models.CharField(max_length=50, null=True)
+    user_image   = models.CharField(max_length=2000, null=True)
```

Comment:
```suggestion
    image_url   = models.URLField(max_length=2000, null=True)
```
강제성은 없지만 URLField 라는 것도 있습니다. 가독성을 위해 사용해주세요.
테이블 명에 해당하는 단어를 컬럼에 중복해서 넣을 필요는 없을 것 같습니다.

Reply:
넵 수정했습니다. 감사합니다.